### ISSUE TITLE
Task - Colour contrast issues

### ISSUE THREAD

**Issue by gggrant** [GCTC-NTGC/TalentCloud#5418]
To get that sweet 100% score:

Accessibility
https://talent.canada.ca/en/jobs ✗ color-contrast - Background and foreground colors do not have a sufficient contrast ratio.
  Element has insufficient color contrast of 2.45 (foreground color: #969696, background color: #eaeaea, font size: 23.4pt, font weight: normal). Expected contrast ratio of 3:1
   `<h2 data-c-font-size=“h2” data-c-color=“gray” data-c-margin=“bottom(1)“>We’re improving the platform. </h2>`

Accessibility
https://talent.canada.ca/fr/jobs ✗ color-contrast - Background and foreground colors do not have a sufficient contrast ratio.
  Element has insufficient color contrast of 2.45 (foreground color: #969696, background color: #eaeaea, font size: 23.4pt, font weight: normal). Expected contrast ratio of 3:1
   `<h2 data-c-font-size=“h2” data-c-color=“gray” data-c-margin=“bottom(1)“>Nous améliorons la plate-forme.</h2>`

Accessibility
https://talent.canada.ca/en/register ✗ color-contrast - Background and foreground colors do not have a sufficient contrast ratio.
  Element has insufficient color contrast of 3.99 (foreground color: #ff0000, background color: #ffffff, font size: 9.6pt, font weight: normal). Expected contrast ratio of 4.5:1
   `<span>Required</span>`
   `<span>Required</span>`
   `<span>Required</span>`
   `<span>Required</span>`

Accessibility
https://talent.canada.ca/en/login ✗ color-contrast - Background and foreground colors do not have a sufficient contrast ratio.
  Element has insufficient color contrast of 3.99 (foreground color: #ff0000, background color: #ffffff, font size: 9.6pt, font weight: normal). Expected contrast ratio of 4.5:1
   `<span>Required</span>`
   `<span>Required</span>`

Accessibility
https://talent.canada.ca/fr/login ✗ color-contrast - Background and foreground colors do not have a sufficient contrast ratio.
  Element has insufficient color contrast of 3.99 (foreground color: #ff0000, background color: #ffffff, font size: 9.6pt, font weight: normal). Expected contrast ratio of 4.5:1
   `<span>Champs obligatoires</span>`
   `<span>Champs obligatoires</span>`

Accessibility
https://talent.canada.ca/fr/register ✗ color-contrast - Background and foreground colors do not have a sufficient contrast ratio.
  Element has insufficient color contrast of 3.99 (foreground color: #ff0000, background color: #ffffff, font size: 9.6pt, font weight: normal). Expected contrast ratio of 4.5:1
   `<span>Champs obligatoires</span>`
   `<span>Champs obligatoires</span>`
   `<span>Champs obligatoires</span>`
   `<span>Champs obligatoires</span>`

Accessibility
https://talent.canada.ca/en/manager/login ✗ color-contrast - Background and foreground colors do not have a sufficient contrast ratio.
  Element has insufficient color contrast of 3.99 (foreground color: #ff0000, background color: #ffffff, font size: 9.6pt, font weight: normal). Expected contrast ratio of 4.5:1
   `<span>Required</span>`
   `<span>Required</span>`

Accessibility
https://talent.canada.ca/en/manager/register ✗ color-contrast - Background and foreground colors do not have a sufficient contrast ratio.
  Element has insufficient color contrast of 3.99 (foreground color: #ff0000, background color: #ffffff, font size: 9.6pt, font weight: normal). Expected contrast ratio of 4.5:1
   `<span>Required</span>`
   `<span>Required</span>`
   `<span>Required</span>`
   `<span>Required</span>`

Accessibility
https://talent.canada.ca/fr/password/reset ✗ color-contrast - Background and foreground colors do not have a sufficient contrast ratio.
  Element has insufficient color contrast of 3.99 (foreground color: #ff0000, background color: #ffffff, font size: 9.6pt, font weight: normal). Expected contrast ratio of 4.5:1
   `<span>Required</span>`

Accessibility
https://talent.canada.ca/fr/manager/register ✗ color-contrast - Background and foreground colors do not have a sufficient contrast ratio.
  Element has insufficient color contrast of 3.99 (foreground color: #ff0000, background color: #ffffff, font size: 9.6pt, font weight: normal). Expected contrast ratio of 4.5:1
   `<span>Champs obligatoires</span>`
   `<span>Champs obligatoires</span>`
   `<span>Champs obligatoires</span>`
   `<span>Champs obligatoires</span>`

Accessibility
https://talent.canada.ca/en/manager/password/reset ✗ color-contrast - Background and foreground colors do not have a sufficient contrast ratio.
  Element has insufficient color contrast of 3.99 (foreground color: #ff0000, background color: #ffffff, font size: 9.6pt, font weight: normal). Expected contrast ratio of 4.5:1
   `<span>Required</span>`

Lighthouse Summary
 Total Pages Scanned: 27
 Total Auditing Time: 246941 ms
 Average Page Audit Time: 43435 ms
 Total Audits Passed: 853 ✓
 Total Violations:
  Accessibility: 26 ✗

**Comment by gggrant** [comment]
Also, on the homepage:
<img width="629" alt="Screen Shot 2021-03-11 at 4 29 34 PM" src="https://user-images.githubusercontent.com/31678168/110857587-0752e100-8287-11eb-8ef2-ccb8699d33c8.png">

**Comment by mnigh** [comment]
On every page, _Date Modified_ text in footer does not have a sufficient colour contrast ratio.

![Screen Shot 2021-03-12 at 11 28 54](https://user-images.githubusercontent.com/3046459/110970401-b6df9000-8327-11eb-87fa-1dc2f60e6137.png)

**Comment by mnigh** [comment]
On Applications index page, _Apply By:_ text does not have a sufficient colour contrast ratio.

![Screen Shot 2021-03-12 at 11 33 44](https://user-images.githubusercontent.com/3046459/110970822-1dfd4480-8328-11eb-8347-87f9720b763a.png)

**Comment by mnigh** [comment]
On Settings page, assorted text does not have a sufficient colour contrast ratio.
![Screen Shot 2021-03-12 at 11 51 40](https://user-images.githubusercontent.com/3046459/110971879-65d09b80-8329-11eb-844f-e68acae717a8.png)

**Comment by mnigh** [comment]
On Job Poster page, assorted text does not have a sufficient colour contrast ratio.
![Screen Shot 2021-03-12 at 11 55 20](https://user-images.githubusercontent.com/3046459/110972352-e68f9780-8329-11eb-8770-ffae36b80274.png)

**Comment by mnigh** [comment]
On Application Timeline Experience page, assorted text does not have a sufficient colour contrast ratio.
![Screen Shot 2021-03-12 at 12 01 35](https://user-images.githubusercontent.com/3046459/110973068-bf859580-832a-11eb-9a39-cae9885c5245.png)

**Comment by mnigh** [comment]
On Application Timeline Skills page, assorted text does not have a sufficient colour contrast ratio.
![Screen Shot 2021-03-12 at 12 06 04](https://user-images.githubusercontent.com/3046459/110973541-5eaa8d00-832b-11eb-9faf-1bb8d47c878c.png)

**Comment by mnigh** [comment]
On Application Timeline Fit page, required text does not have a sufficient colour contrast ratio.
![Screen Shot 2021-03-12 at 12 12 29](https://user-images.githubusercontent.com/3046459/110974222-4d15b500-832c-11eb-879b-3d3955aaef13.png)

**Comment by mnigh** [comment]
On Application Timeline Submission page, required text does not have a sufficient colour contrast ratio.
![Screen Shot 2021-03-12 at 12 14 06](https://user-images.githubusercontent.com/3046459/110974405-7e8e8080-832c-11eb-8a51-889caef6ecef.png)
